pkg/restore/pod_action_test.go

```diff
@@ -169,17 +170,23 @@ func TestPodActionExecute(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			action := NewPodAction(velerotest.NewLogger())
 
-			res, warning, err := action.Execute(test.obj, nil)
-
-			assert.Nil(t, warning)
+			res, err := action.Execute(&RestoreItemActionExecuteInput{
+				Item:           test.obj,
+				ItemFromBackup: test.obj,
+				Restore:        nil,
+			})
 
 			if test.expectedErr {
 				assert.NotNil(t, err, "expected an error")
-			} else {
-				assert.Nil(t, err, "expected no error, got %v", err)
+				return
 			}
+			assert.Nil(t, err, "expected no error, got %v", err)
 
-			assert.Equal(t, test.expectedRes, res)
+			if !test.expectedErr {
```

Comment:
this `if` can now be dropped since you're returning early if `test.expectedErr == true`

Reply:
`if` removed